packages/sdk-middleware-auth/test/build-requests.spec.js

```diff
@@ -31,6 +31,28 @@ describe('buildRequestForClientCredentialsFlow', () => {
     })
   })
 
+  it('parses a host that ends with \'/\'', () => {
```

Comment:
To make this more readable I would prefer you use backtick _`_ and quotes to emphasise the forward slash